robots/so_arm100/so_arm100.srdf.xacro

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="so_arm100">
+  <disable_collisions link1="Base" link2="Rotation_Pitch" reason="Adjacent" />
+  <disable_collisions link1="Lower_Arm" link2="Upper_Arm" reason="Adjacent" />
+  <disable_collisions link1="Lower_Arm" link2="Wrist_Pitch_Roll" reason="Adjacent" />
+  <disable_collisions link1="Moving_Jaw" link2="Fixed_Jaw" reason="Adjacent" />
+  <disable_collisions link1="Rotation_Pitch" link2="Upper_Arm" reason="Adjacent" />
+  <disable_collisions link1="Wrist_Pitch_Roll" link2="Fixed_Jaw" reason="Adjacent" />
```

Comment:
Maybe this robot could be a good example on creating a robot model without configs.toml?